meshtastic/node_runtime/content_runtime.py

```
@@ -0,0 +1,496 @@
+                has_ringtone_response = False
+        if raw_admin is None or not has_ringtone_response:
+            logger.warning("Unexpected ringtone response without raw ringtone data")
+            return (True, None)
+        try:
+            ringtone_part = raw_admin.get_ringtone_response
+            return (True, ringtone_part)
+        except AttributeError:
+            logger.warning("Failed to parse ringtone response payload")
+            return (True, None)
+
+    def handle_canned_message_response(
```

Ruff (0.15.6)

[warning] Too many return statements (7 > 6)

(PLR0911)